screen-attributions/src/main/java/com/ivy/attributions/AttributionsScreen.kt

```diff
@@ -0,0 +1,190 @@
+import com.ivy.design.l0_system.Gray
+import com.ivy.design.l0_system.UI
+import com.ivy.design.l0_system.style
+import com.ivy.legacy.IvyWalletPreview
+import com.ivy.navigation.Navigation
+import com.ivy.navigation.navigation
+
+@Composable
+fun AttributionsScreenImpl() {
+    val viewModel: AttributionsViewModel = viewModel()
+    val uiState = viewModel.uiState()
+
+    AttributionsUI(uiState = uiState)
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun AttributionsUI(
+    uiState: AttributionsState
+) {
+    val nav = navigation()
+
+    Scaffold(
+        topBar = {
+            TopAppBar(
+                title = {
+                    TopAppBarTitle(text = "Attributions")
+                },
+                navigationIcon = {
+                    BackButton(nav = nav)
+                }
+            )
+        },
+        content = {
+            AttributionsContent(paddingValues = it, attributionItems = uiState.attributionItems)
+        }
+    )
+}
+
+@Composable
+private fun TopAppBarTitle(text: String) {
+    Text(
+        text = text,
+        maxLines = 1,
+        overflow = TextOverflow.Ellipsis,
+        style = UI.typo.h2.style(
+            fontWeight = FontWeight.Black
+        )
+    )
+}
+
+@Composable
+private fun BackButton(nav: Navigation) {
+    IconButton(onClick = {
+        nav.back()
+    }) {
+        Icon(
+            imageVector = Icons.Filled.ArrowBack,
+            contentDescription = "Back"
+        )
+    }
+}
+
+@Composable
+private fun AttributionsContent(
+    paddingValues: PaddingValues,
+    attributionItems: List<AttributionItem>
+) {
+    LazyColumn(
+        modifier = Modifier
+            .padding(paddingValues)
+            .padding(horizontal = 16.dp, vertical = 4.dp),
+        verticalArrangement = Arrangement.spacedBy(8.dp)
+    ) {
+        items(attributionItems) { attributionItem ->
+            AttributionLayout(attributionItem)
+        }
+    }
+}
+
+@Composable
+private fun AttributionLayout(
+    attributionItem: AttributionItem
+) {
+    when (attributionItem) {
+        is AttributionItem.Attribution -> AttributionCard(attribution = attributionItem)
+        is AttributionItem.Divider -> AttributionsSectionDivider(text = attributionItem.sectionName)
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun AttributionCard(attribution: AttributionItem.Attribution) {
+    val browser = LocalUriHandler.current
+
+    Card(
+        shape = RoundedCornerShape(12.dp),
+        onClick = {
+            browser.openUri(attribution.link)
+        }
+    ) {
+        Column(
+            modifier = Modifier.padding(horizontal = 12.dp, vertical = 8.dp)
+        ) {
+            Text(
+                text = attribution.name
+            )
+        }
+    }
+}
+
+@Composable
+private fun AttributionsSectionDivider(
+    text: String,
+    color: Color = Gray
+) {
+    Spacer(modifier = Modifier.height(12.dp))
```

Comment:
The Spacer must not be part of the Attribution divider. Detekt will complain and tell you why